crates/testing/tests/tests_1/test_success.rs

```diff
@@ -39,6 +42,31 @@ cross_tests!(
     },
 );
 
+cross_tests!(
```

Comment:
Probably would make sense to also add test for 0.1 -> 0.2 upgrade

Reply:
Nothing in consensus is gated on the fee upgrade (or even has access to a constant representing its version number), so I think the marketplace test passing without any view failures already covers general upgrades too.

Reply:
Makes sense, thanks